### PR TITLE
Array Test Task.

### PR DESCRIPTION
**Pull request by AlexanderVladimirov9090**
Done.

### CODE REVIEW

**Comment by mgenov** [comment]
PR contains 1831 lines. Maybe you have added something that is not for review.

**Comment by AlexanderVladimirov9090** [comment]
Yes I see I`v exported The outcome of test in HTML and it generated some stuff. Will remove it A.S.A.P.

File: test/com/clouway/arraytask/ArrayTest.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  }
+
+  @Test
+  public void getMinOneElementArray() {
+    int result = array.getMinElement(arrOneElement);
+    assertEquals(0, result, 0);
+  }
+
+  @Test
+  public void getMaxOneElementArray() {
```
Comment (mgenov) [inline test/com/clouway/arraytask/ArrayTest.java]: For such simple things It is much more readable to set up test data inline. 

```java
assertThat(new Array().getMinElement(new int[] {10, 12, 5, 6}),  is(equalTo(5)));
```

File: test/com/clouway/arraytask/ArrayTest.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    array.getMinElement(emptyArray);
+  }
+
+  @Test(expected = ArrayIndexOutOfBoundsException.class)
+  public void getMaxElement() throws Exception {
+    array.getMaxElement(emptyArray);
+  }
+
+  @Test
+  public void printArray() throws Exception {
```
Comment (mgenov) [inline test/com/clouway/arraytask/ArrayTest.java]: I'm not sure what you are verifying here.

Comment (mgenov) [inline test/com/clouway/arraytask/ArrayTest.java]: printArray method is not doing anything currently. In this test you have to check that all elements are printed.

**Pull request closed without merging**